src/goddard/renderer.c

```diff
@@ -2693,7 +2702,7 @@ void gd_setproperty(enum GdProperty prop, f32 f1, f32 f2, f32 f3) {
 }
 
 /* 2522B0 -> 2522C0 */
-void stub_renderer_5(void) {
+UNUSED void stub_renderer_5(void) {
```

Comment:
the funny empty function (there's too many of them for me to bother commenting on all of them)